src/app/clusters/ias-zone-client/ias-zone-client.cpp

```diff
@@ -390,7 +390,8 @@ static void removeServer(uint8_t * ieeeAddress)
 static EmberStatus sendCommand(EmberNodeId destAddress)
 {
     emberAfSetCommandEndpoints(myEndpoint, emberAfIasZoneClientKnownServers[currentIndex].endpoint);
-    EmberStatus status = emberAfSendCommandUnicast(EMBER_OUTGOING_DIRECT, destAddress);
+    MessageSendDestination destination = MessageSendDestination(MessageSendDestination::OutGoingDirect(destAddress));
```

Comment:
How does this line even compile?  There is no `MessageSendDestination::OutGoingDirect`.

Similar in various other places in this diff.

Reply:
Done. It looks like this files is not compiled.